### ISSUE TITLE
`diamond` object-class throws syntax error

### ISSUE THREAD

**Issue by dlo** [notlibrary/obsidian-adamantine-pick#1]
This plugin is great, thank you! Today I found a issue when attempting to render the following diagram:

  \```pikchr
  diamond
  \```

This throws the following error:

```
/*    1 */  diamond
                  ^
ERROR: syntax error
```

Any ideas what might be going on here? Thanks in advance!

**Comment by notlibrary** [comment]
Old source artifact fixed that in 0.5.2 plugin version update it if you will
![image](https://github.com/notlibrary/obsidian-adamantine-pick/assets/40695473/0763665e-990f-42c6-b6f5-aee18b27ad94)

**Comment by dlo** [comment]
Incredible! Thank you!
